synapse/cmds/trigger.py

```diff
@@ -134,7 +134,8 @@ def _make_argparser(self):
         subparsers = parser.add_subparsers(title='subcommands', required=True, dest='cmd',
                                            parser_class=functools.partial(s_cmd.Parser, outp=self))
 
-        subparsers.add_parser('list', help="List triggers you're allowed to manipulate", usage=ListHelp)
+        parser_list = subparsers.add_parser('list', help="List triggers you're allowed to manipulate", usage=ListHelp)
```

Comment:
Do we want to make any changes to the `cmdr` commands at all?
The storm interfaces are preferred now.

Reply:
No you're right. I just missed the deprecation warning at the bottom. Best to leave this one alone and usher people to the storm interfaces if they want any of the new stuff.